File: Week01/src/homework1/Three.java
package homework1;


public class Three {

	public static void main(String[] args) {
		for (int i = 0; i <3 ; i++) {
			
			for (int k = 0; k < 2-i;  k++) {		// 공백찍기
			System.out.print(" ");
			}
			
			for (int j = 0; j<i+1; j++) {			// 별찍기
				System.out.print("*");
			}
			System.out.println();					// 쥴버뀸
		
	}


}
}

File: ThreeJS/my first threejs/src/js/three.js
import * as THREE from 'three';
import { renderer, scene, camera } from './three/renderCamera.js';
import { loaderAnimation } from './three/loader.js';
import { } from './three/text.js';

function animate() {
  loaderAnimation();
  
  requestAnimationFrame(animate);
  renderer.render(scene, camera);
}
animate();
